libs/core/src/lib/calendar/calendar-header/calendar-header.component.ts

```diff
@@ -24,6 +24,12 @@ import { CalendarCurrent } from '../models/calendar-current';
 import { CalendarYearGrid } from '../models/calendar-year-grid';
 import { CalendarService } from '../calendar.service';
 
+export type NavigationButtonDisableFunction<D> = (
```

Comment:
why this type is declared inside header component, where it's not used? Let's move it to separate file in models folder